common/src/main/java/whocraft/tardis_refined/client/screen/selections/ShellSelectionScreen.java

```diff
@@ -111,6 +139,15 @@ public void render(PoseStack poseStack, int i, int j, float f) {
         /*Render Widgets*/
         super.render(poseStack, i, j, f);
 
+        double alpha = (100.0D - this.age * 3.0D) / 100.0D;
```

Comment:
Would be good to add some general comments about these magic numbers in case we forget later on.